common/components/ConfirmationModalTemplate/components/Body/Body.tsx

```diff
@@ -0,0 +1,119 @@
+import { Addresses } from './components/Addresses';
+import { Amounts } from './components/Amounts';
+import { Details } from './components/Details';
+import React from 'react';
+import { SerializedTransaction } from 'components/renderCbs';
+import { makeTransaction, getTransactionFields } from 'libs/transaction';
+import { connect } from 'react-redux';
+import { AppState } from 'reducers';
+import { getFrom, getUnit, getDecimal, getTo } from 'selectors/transaction';
+import { getNetworkConfig, getNodeConfig } from 'selectors/config';
+import { getTransactionFee } from 'libs/transaction/utils/ether';
+import { Wei, TokenValue } from 'libs/units';
+import ERC20 from 'libs/erc20';
+import './Body.scss';
+
+interface State {
+  showDetails: boolean;
+}
+
+interface Props {
```

Comment:
Typing up props this way makes them fragile to State/Selector changes. Any changes to the selectors / state will break these typings as they dont reference `AppState`

Reply:
Also, re-naming it as `StateProps` will help readability to tell where the props are coming from.